docs/developing/config.md

```diff
@@ -202,19 +101,6 @@ class AppComponent {
 | **Description** | Returns a config value as a `number`. Returns `0` if the config is not defined. |
 | **Signature**   | `getNumber(key: string, fallback?: number) => number`                             |
 
-#### Examples
-
-```ts
-import { Config } from '@ionic/angular';
-
-@Component(...)
-class AppComponent {
-  constructor(config: Config) {
-    const keyboardHeight = config.getNumber('keyboardHeight');
-  }
-}
-```
-
```

Comment:
Was this example removed intentionally? I know the info is fairly redundant, but it seems strange to have an example on the other two but not this one. Maybe one example for the whole provider would be easier?

Reply:
This was moved to `config#getnumber`. React and Vue currently do not provide this kind of API which is why I made it Angular-only.